font-codegen/Cargo.toml

```diff
@@ -24,3 +24,4 @@ xflags = "0.2.4"
 log = "0.4"
 env_logger = "0.9.0"
 rayon = "1.5.3"
+indexmap = "1.9.1"
```

Comment:
I suggest that adding a dependency is a sufficiently large change it should be explained in comments. For example, why can't we use the normal HashMap?

Reply:
good point: the motivation here is that hashmap does not guarantee ordering, and we want to generate code for our items in the order that they are declared, otherwise any rerun of codegen risks shuffling everything.